Networking/NetworkingTests/Remote/MediaRemoteTests.swift

```diff
@@ -17,6 +17,41 @@ final class MediaRemoteTests: XCTestCase {
         network.removeAllSimulatedResponses()
     }
 
+    // MARK: - Load Media From Media Library `loadMediaLibrary`
+
+    /// Verifies that `loadMediaLibrary` properly parses the `media-library` sample response.
+    ///
+    func testLoadMediaLibraryProperlyReturnsParsedMedia() {
+        let remote = MediaRemote(network: network)
+        let expectation = self.expectation(description: "Load Media Library")
+
+        network.simulateResponse(requestUrlSuffix: "media", filename: "media-library")
+
+        remote.loadMediaLibrary(for: sampleSiteID) { mediaItems, error in
```

Comment:
yea it'd be nice to test the API response with various params, though it doesn't feel very scalable with the mock API response in our Networking tests. Right now, we don't call the API in the Networking tests (to avoid network errors, server flakiness etc.), and we manually generate the mock API response by making the API call like in Postman. I'm concerned that the manual API response generation could be time consuming (I replaced the domain before adding the response JSON to the codebase) and prone to human errors, so I think keeping the basic API response validation is sufficient here considering the maintenance cost.

I'd love to hear if you have any thoughts on testing the API more thoroughly while balancing the maintenance effort!

Reply:
Yeah, you're right. It's best to just keep this as JSON response testing for now. 

In the future, we can probably have a separate test suite that runs tests against the production API. We can run that periodically. Platform is working on something similar so we can wait and see how that works.